Makefile

```diff
@@ -69,14 +69,22 @@ export LAUNCHER_PROGRAM_EXT
 # See https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
 .PHONY: build bundle bundle-msi test copy-test-files generate clean sign dist help
 
+build: generate compile compress package  ## Build (default)
+
 # Default target
-build: generate  ## Build (default)
+compile: generate  ## Compile with go build. Run after make generate.
```

Comment:
The ugly Make syntax 'compile: generate' syntax means, that it calls generate before. So changing the comment here is not really needed.

Reply:
We now use build by default.